### PR TITLE
Fix travel insurance widget icons and autocomplete

### PR DESCRIPTION
**Pull request by remybach**

### CODE REVIEW

File: app/assets/stylesheets/widgets/_travel_insurance.sass [hunk 1]
```diff
@@ -55,6 +55,14 @@
     float: left
     margin: 5px 5px 10px 5px
 
+  .icon
+    +size(22px)
+    background-position: 0 20%
+    background-size: cover
+    display: inline-block
```
Comment (Ianfeather) [inline app/assets/stylesheets/widgets/_travel_insurance.sass]: should be `@extend  %inline-block`

File: app/assets/stylesheets/widgets/_travel_insurance.sass [hunk 2]
```diff
@@ -55,6 +55,14 @@
...
+    float: right
+    margin: -30px 5px 0 0
```
Comment (Ianfeather) [inline app/assets/stylesheets/widgets/_travel_insurance.sass]: More magic ;)

**Comment by Ianfeather** [comment]
Looks fine, minor comments. +1

File: app/assets/stylesheets/widgets/_travel_insurance.sass (hunk 1 above)
Comment (remybach) [inline app/assets/stylesheets/widgets/_travel_insurance.sass]: Good catch :+1:

File: app/assets/stylesheets/widgets/_travel_insurance.sass (hunk 2 above)
Comment (remybach) [inline app/assets/stylesheets/widgets/_travel_insurance.sass]: Yeah, I don't have much control over the markup that's returned here as it's generated by third party JS :confused:

**Comment by Ianfeather** [comment]
+1 Check the failing build though

**Pull request merged**